RELEASE-NOTES.md

```diff
@@ -183,7 +187,6 @@ Changes in the `Entity` hierarchy:
 * Changed the constructor signature of `Property`
 * Removed `Entity::setClaims` (`Item::setClaims` has been retained)
 * Removed `Entity::stub`
-* Removed `Property::newEmpty`
```

Comment:
I checked and this line is just not true. The method moved, that's why it may look like it's removed in diffs.